app/components/warning-message.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  actions: {
+    setIsClosed() {
```

Comment:
You are manually manipulating the DOM in this action handler, which isn't a good practice in Ember. This is because Ember expects to control the DOM, and when you manually mutate it, the state of the DOM gets out of sync with what Ember thinks it is.
You already have the `isClosed` property in the controller, and are conditionally wrapping the warning-message on it, you should fully utilize it.